dbcon/joblist/CMakeLists.txt

```diff
@@ -36,7 +36,6 @@ set(joblist_LIB_SRCS
     pcolstep.cpp
     pdictionary.cpp
     pdictionaryscan.cpp
-    primitivemsg.cpp
```

Comment:
I believe this source file is redundant now, so should be removed from the codebase.

Reply:
removed